lib/Helper/DataSetUploadHandler.php

```diff
@@ -62,6 +62,20 @@ protected function handleFormData($file, $index)
                 throw new AccessDeniedException();
             }
 
+            // Get all columns
+            $columns = $dataSet->getColumn();
+
+            // Filter columns where dataSetColumnType is "Value"
+            $filteredColumns = array_filter($columns, function ($column) {
+                return $column->dataSetColumnType == 'Value';
```

Comment:
Might be better to rely on `dataSetColumnTypeId` instead? ( result should be the same )